flight/Modules/UAVOMSPBridge/3rdparty/serial_4way.c

```diff
@@ -76,7 +76,7 @@
 // *** change to adapt Revision
 #define SERIAL_4WAY_VER_MAIN 20
 #define SERIAL_4WAY_VER_SUB_1 (uint8_t) 0
-#define SERIAL_4WAY_VER_SUB_2 (uint8_t) 00
+#define SERIAL_4WAY_VER_SUB_2 (uint8_t) 01
```

Comment:
This is not ideal; the version was bumped after no functional change-- a couple of months after blheli32 support was added.  Then Blhelisuite moved to depend upon the new version.